sidebars.js

```diff
@@ -16,6 +16,13 @@ module.exports = {
     Build: [
       "infra_testnet"
     ],
+    Docs: [
```

Comment:
I think it is confusing to have a section called "Docs" in the docs 

can you put it as a direct link `Rust Docs` instead? What I mean is without the expandable dropdown

Reply:
I tried, but I couldn't find a way how to achieve this result

Reply:
alternatively you can move it as first link under section Build